code/controllers/subsystems/atoms.dm

```diff
@@ -48,15 +48,18 @@ SUBSYSTEM_DEF(atoms)
 				++count
 				CHECK_TICK
 
-	report_progress("Initialized [count] atom\s")
+	if(!Master.current_runlevel)//So it only does it at roundstart
+		report_progress("Initialized [count] atom\s")
```

Comment:
All of that does remove the spam, but I think it just hides the problem. The actual problem being: every single maploaded chunk causes its own set of initializations, rebuilds atmos and powernets, etc.

Reply:
Check the dungeon generator code, my dude. I actually did that.

Reply:
code/modules/dungeons/crawler/dungeon_gen.dm lines 301 and further, @ACCount12